ui/urls.py

```diff
@@ -23,10 +23,13 @@
 
     url(r'^videos/(?P<video_key>[0-9a-f]{32})/$', views.VideoDetail.as_view(), name='video-detail'),
     url(r'^videos/(?P<video_key>[0-9a-f]{32})/embed/$', views.VideoEmbed.as_view(), name='video-embed'),
+    url(r'^videos/(?P<video_key>[0-9a-f]{32})/download', views.VideoDownload.as_view(), name='video-download'),
```

Comment:
Can we add a `/$` or `/?$` at the end here and below for consistency?

Reply:
The URL needs to handle possible variations like `/download.mp4` or `/download.qt`